head/src/test/kotlin/io/qalipsis/core/head/model/converter/CampaignConfigurationConverterImplTest.kt

```diff
@@ -138,10 +175,15 @@ internal class CampaignConfigurationConverterImplTest {
                 )
             )
         )
+
+        coVerifyOrder {
+            hook1.preCreate(request, any())
```

Comment:
```suggestion
            hook1.preCreate(ref(request), ref(result))
```
Apply the same everywhere.